mipengine/common/validate_identifier_names.py

```diff
@@ -6,15 +6,15 @@ def validate_identifier_names(func):
     wraps(func)
 
     def wrapper(*args, **kwargs):
-        all_args = list(args) + list(kwargs.values())
-        for arg in all_args:
-            if type(arg) == str:
-                if not arg.isidentifier() and not arg.isalnum() and not has_proper_url_format(arg):
-                    raise ValueError(f"Not allowed character in argument: {arg}")
-            elif type(arg) == list:
-                for item in arg:
-                    if not item.isidentifier() and not item.isalnum() and not has_proper_url_format(item):
-                        raise ValueError(f"Not allowed character in argument: {item}")
+        # all_args = list(args) + list(kwargs.values())
```

Comment:
Git does not behave well when merging commented out sections.

If possible we should avoid this.